bundles/framework/publisher2/handler/ToolPanelHandler.js

```diff
@@ -39,8 +39,7 @@ class UIHandler extends StateHandler {
                     component: toolComponent.component,
                     handler: toolComponent.handler,
                     publisherTool: tool,
-                    id: tool.getTool().id,
-                    title: tool.getTool().title
+                    ...tool.getTool()
```

Comment:
We should just add `tool.hideCheckbox` here instead so it's clear (for maintenance purposes) what is/can be passed.

Reply:
Object spread -> explicit set.